system-parachains/bridge-hubs/bridge-hub-polkadot/src/lib.rs

```diff
@@ -417,7 +419,7 @@ impl pallet_aura::Config for Runtime {
 	type MaxAuthorities = ConstU32<100_000>;
 	type AllowMultipleBlocksPerSlot = ConstBool<false>;
 	#[cfg(feature = "experimental")]
```

Comment:
This feature is not enabled. 

We should really remove this feature, here and in polkadot-sdk. For now we can remove it here and enable for the crates directly.

Reply:
Do you mean to remove its usage from the code or get rid of it completely in manifests?

Reply:
For this repo you should remove the feature from the code entirely. Enable the feature by default for the crate. 

In Polkadot-sdk, get rid off the feature completely. It was added randomly to guard something now, it is there while all the other code changes of async backing just got merged as is.

Reply:
Please check the last commit, hope I got it right.